### PR TITLE
fix(select): remove incorrect props (FE-4983)

### PR DESCRIPTION
**Pull request by grabkowski**
### Proposed behaviour

remove props added by a bad merge

### Current behaviour

incorrect props are triggering AXE warnings

### Checklist

- [x] Commits follow our style guide
- [x] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [ ] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.

### CODE REVIEW

**Review by Parsium — APPROVED** [review]

File: src/components/select/filterable-select/filterable-select.component.js [hunk 1]
```diff
@@ -525,16 +525,13 @@ const FilterableSelect = React.forwardRef(
...
-        aria-expanded={isOpen}
-        aria-haspopup="listbox"
         ref={containerRef}
         hasTextCursor
         readOnly={readOnly}
         disabled={disabled}
         data-component={dataComponent}
         data-role={dataRole}
         data-element={dataElement}
-        isOpen={isOpen}
```
Comment (mkrds) [inline src/components/select/filterable-select/filterable-select.component.js:537]: `isOpen` is required here

File: src/components/select/multi-select/multi-select.component.js [hunk 2]
```diff
@@ -520,16 +520,13 @@ const MultiSelect = React.forwardRef(
...
-        aria-expanded={isOpen}
-        aria-haspopup="listbox"
         disabled={disabled}
         readOnly={readOnly}
         hasTextCursor
         size={size}
         data-component={dataComponent}
         data-role={dataRole}
         data-element={dataElement}
-        isOpen={isOpen}
```
Comment (mkrds) [inline src/components/select/multi-select/multi-select.component.js:532]: `isOpen` is required here

File: src/components/select/simple-select/simple-select.component.js [hunk 3]
```diff
@@ -416,12 +416,9 @@ const SimpleSelect = React.forwardRef(
         transparent={transparent}
         disabled={disabled}
         readOnly={readOnly}
-        aria-expanded={isOpen}
-        aria-haspopup="listbox"
         data-component={dataComponent}
         data-role={dataRole}
         data-element={dataElement}
-        isOpen={isOpen}
```
Comment (mkrds) [inline src/components/select/simple-select/simple-select.component.js:424]: `isOpen` is required here

Comment (grabkowski) [reply src/components/select/simple-select/simple-select.component.js:424]: added

File: src/components/select/multi-select/multi-select.component.js (hunk 2 above)
Comment (grabkowski) [reply src/components/select/multi-select/multi-select.component.js:532]: added

File: src/components/select/filterable-select/filterable-select.component.js (hunk 1 above)
Comment (grabkowski) [reply src/components/select/filterable-select/filterable-select.component.js:537]: added

**Review by mkrds — DISMISSED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by Parsium — DISMISSED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by mkrds — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Pull request merged**